internal/exec/stages/files/filesystemEntries.go

```diff
@@ -142,6 +142,7 @@ func (s *stage) createResultFile() error {
 	s.Logger.PushPrefix("createResultFile")
 	defer s.Logger.PopPrefix()
 
+	s.relabel(distro.ResultFilePath())
```

Comment:
This should already be handled by `s.createEntries()`.

Reply:
Yeah, agreed.